flink-autoscaler/src/main/java/org/apache/flink/autoscaler/state/AutoScalerStateStore.java

```diff
@@ -69,6 +70,13 @@ void storeParallelismOverrides(Context jobContext, Map<String, String> paralleli
 
     void removeParallelismOverrides(Context jobContext) throws Exception;
 
+    void storeConfigOverrides(Context jobContext, Configuration configOverrides) throws Exception;
```

Comment:
Could we rename it to `config` instead of `configOverrides`?

I guess we might store some other configs into it even if it isn't overrides related. WDYT?

Reply:
The idea is that those are applied on top of the regular config, that's why I chose configOverrides over config.

Reply:
I'm happy to rename but I can't imagine another use right now. Are you suggesting we might store other state in the Configuration which is not related to the configuration? I think that goes in the direction Gyula's comment (https://github.com/apache/flink-kubernetes-operator/pull/762#issuecomment-1926743545).